iceoryx_binding_c/test/moduletests/test_subscriber.cpp

```diff
@@ -340,6 +340,18 @@ TEST_F(iox_sub_test, deinitSubscriberDetachesTriggerFromWaitSet)
     free(subscriber);
 }
 
+TEST_F(iox_sub_test, correctServiceDescriptionReturned)
+{
+    auto serviceDescription = iox_sub_get_service_description(m_sut);
+
+    EXPECT_THAT(serviceDescription.serviceId, Eq(iox::capro::InvalidID));
```

Comment:
Do we have cases were those Ids are set? If not, why do they exist? If yes, it should be tested. In any case: the ServiceDescription needs some redesign.

Reply:
Not yet but I think we have to refactor the c publisher and c subscriber in the near future so that a service description is provided as argument and not three strings. Then it will become a use case